book/controller.rst

```diff
@@ -803,6 +803,29 @@ Just like when creating a controller for a route, the order of the arguments of
 order of the arguments, Symfony will still pass the correct value to each
 variable.
 
+Checking the Validity of a CSRF Token
```

Comment:
Excuse me for nitpicking but `Checking the Validity ...` sounds a bit too formal to me. Should we change this title by `Validating a CSRF Token`?